.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: Testing
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
+  pull_request:
+
+env:
+  _AZ_REGISTRY: "bitwardenprod.azurecr.io"
+
+jobs:
+  testing:
+    name: Run tests
+    runs-on: ubuntu-22.04
```

Comment:
❓ Does this need to be tested on windows and *nix? I'm thinking mostly about path manipulation, fwiw, but it could also be important if we're running integration tests for IIS hosts.